.github/workflows/sklearn.yml

```diff
@@ -0,0 +1,30 @@
+name: sklearn
+
+# DELETE THIS FILE LATER
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  sklearn:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version: ["0.20.3", "0.23.2"]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: "3.6"
+      - name: Install dependencies
+        run: |
+          pip install -e .
+          pip install -r dev/small-requirements.txt
+          pip install scikit-learn==${{ matrix.version }}
+      - name: Run tests
+        run: |
+          pytest --verbose tests/sklearn/test_sklearn_autolog.py
```

Comment:
Created this file to iterate faster and compare results between old and new sklearns. I'll delete this before merging.